api/exclusions/exclusionService.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Response } from 'express';
+import { EnhancedRequest } from '../lib/models';
+
+export async function getUserExclusions(req: EnhancedRequest, res: Response, next: NextFunction) {
+    const isJudge = req.session.passport.user.userinfo.roles.includes('caseworker-ia-iacjudge');
+    let exclusions;
+    if (isJudge) {
+        exclusions = [
+            {
+                added: Date.UTC(2021, 7, 1),
```

Comment:
Should email be in here?

Reply:
no

Reply:
as discussed with Uday, the 'dumb' component requires the email address so it has been agreed that Uday will add it to the RoleExclusion model.